e2e/_suites/ingest-manager/features/ingest-manager.feature

```diff
@@ -0,0 +1,46 @@
+@ingest
```

Comment:
It's possible to add annotations in the form of tags. We can then instrument the test runner to filter by tags, even combining multiple of them:

```shell
godog -t "ingest && unenroll"
```